index.html

```diff
@@ -154,18 +154,18 @@
 <a>Decentralized identifiers</a> (DIDs) are a new type of identifier that
 enables verifiable, decentralized digital identity. A <a>DID</a> identifies
 any subject (e.g., a person, organization, thing, data model, abstract entity, etc.)
-that the controller of the <a>DID</a> decides that it 
-identifies. These new identifiers are designed to enable the controller 
-of a <a>DID</a> to prove control over it and to be implemented independently 
-of any centralized registry, identity provider, or certificate authority. 
-<a>DID</a>s are URLs that associate a <a>DID subject</a> with a 
-<a>DID document</a> allowing trustable interactions associated with that subject. 
-Each <a>DID document</a> can express cryptographic material, verification methods, 
-or <a>service endpoints</a>, which provide a set of mechanisms enabling a 
-<a>DID controller</a> to prove control of the <a>DID</a>. <a>Service 
-endpoints</a> enable trusted interactions associated with the <a>DID 
-subject</a>. A <a>DID document</a> might contain semantics about the subject 
-that it identifies. A <a>DID document</a> might contain the <a>DID subject</a> 
+that the controller of the <a>DID</a> decides that it
+identifies. These new identifiers are designed to enable the controller
+of a <a>DID</a> to prove control over it and to be implemented independently
+of any centralized registry, identity provider, or certificate authority.
+<a>DID</a>s are URLs that associate a <a>DID subject</a> with a
+<a>DID document</a> allowing trustable interactions associated with that subject.
+Each <a>DID document</a> can express cryptographic material, verification methods,
+or <a>service endpoints</a>, which provide a set of mechanisms enabling a
+<a>DID controller</a> to prove control of the <a>DID</a>. <a>Service
+endpoints</a> enable trusted interactions associated with the <a>DID
+subject</a>. A <a>DID document</a> might contain semantics about the subject
+that it identifies. A <a>DID document</a> might contain the <a>DID subject</a>
```

Comment:
Do these need to be in this specific pull request? There are several similar changes here. Space cleanup commits like this should be left to separate pull requests so as not to distract from discussion on content.

Reply:
Yes, agreed, ideally they would be. Merging anyway so that these are fixed when merged into the main document. Will rebase the other PRs so they show up there.